docs/integrations/databases/ClickHouse.mdx

```diff
@@ -88,3 +88,38 @@ However, this may not be optimal for various use cases. Since [table creation](h
 in `ClickHouse` can involve numerous details, it is strongly advised to
 create the destination table manually before loading data to ensure it
 meets specific requirements.
+
+Here is the ClickHouse data exporter code snippet that you can use:
+
+```python
+from mage_ai.settings.repo import get_repo_path
+from mage_ai.io.config import ConfigFileLoader
+from mage_ai.io.clickhouse import ClickHouse
+from pandas import DataFrame
+from os import path
+
+if 'data_exporter' not in globals():
+    from mage_ai.data_preparation.decorators import data_exporter
+
+
+@data_exporter
+def export_data_to_clickhouse(df: DataFrame, **kwargs) -> None:
+    """
+    Template for exporting data to a Clickhouse database.
+    Specify your configuration settings in 'io_config.yaml'.
+
+    """
+    database = 'database_name'  # Specify the name of the schema to export data to
+    table_name = 'table_name'  # Specify the name of the table to export data to
+    config_path = path.join(get_repo_path(), 'io_config.yaml')
+    config_profile = 'default'
+
+    with ClickHouse.with_config(ConfigFileLoader(config_path, config_profile)) as loader:
```

Comment:
I mean you don't need `with` statement.
just do 
```
ClickHouse.with_config(ConfigFileLoader(config_path, config_profile)).export(
...
)
```
Example: https://github.com/mage-ai/mage-ai/blob/master/mage_ai/data_preparation/templates/data_exporters/bigquery.py#L23